src/XeroPHP/Models/PayrollUS/Timesheet/TimesheetLine.php

```diff
@@ -103,7 +103,7 @@ public static function getProperties()
         return [
             'EarningsTypeID' => [false, self::PROPERTY_TYPE_STRING, null, false, false],
             'TrackingItemID' => [false, self::PROPERTY_TYPE_STRING, null, false, false],
-            'NumberOfUnits' => [false, self::PROPERTY_TYPE_STRING, null, false, false],
+            'NumberOfUnits' => [false, self::PROPERTY_TYPE_STRING, null, true, false],
```

Comment:
Hey, unfortunately this is generated code so it'll get overridden.  Do you think you cold update the doc parser to make this change?